config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-0-3.yaml

```diff
@@ -0,0 +1,83 @@
+periodics:
+- name: periodic-cluster-api-test-release-0-3
+  interval: 1h
```

Comment:
I think we can increase the frequency for v0.3 jobs (applies to all)

Reply:
Let's do it in another PR?